### DISCUSSION TITLE
Agent with URL Server "Local", client host can't connect.

### DISCUSSION THREAD

**Discussion by artem33region** [Ylianst/MeshCentral#4908] [Q&A] [answered]
Hi all!
I have meshcentral 1.1.2 in docker.

My config.json file:
`
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "settings": {
    "cert": "my.domain.com",
    "MongoDb": "mongodb://username:password@ip_address:27017/database",
    "dbRecordsEncryptKey": "SyperEncryptKey",
    "WANonly": true,
    "LANonly": true,
    "sessionKey": "SyperSessionKey",
    "port": 443,
    "_aliasPort": 443,
    "redirPort": 80,
    "_redirAliasPort": 80,
    "AgentPong": 300,
    "TLSOffload": false,
    "SelfUpdate": false,
    "AllowFraming": "false",
    "WebRTC": "false"
  },
  "domains": {
      "": {
        "title": "",
        "title2": "",
        "titlepicture": "title-aitek.png",
        "auth": "ldap",
        "ldapUserName": "sAMAccountName",
        "ldapUserKey": "sAMAccountName",
        "ldapOptions": {
          "url": "ldap://my.domain.com",
          "bindDN": "CN=user,CN=my,DC=domain,DC=com",
          "bindCredentials": "password",
          "searchBase": "DC=domain,DC=com",
          "searchFilter": "(sAMAccountName={{username}})"
        },
        "agentCustomization": {
          "displayName": "Remote agent",
          "description": "Agent for remote management",
          "companyName": "Company",
          "serviceName": "meshcentralCompany",
          "fileName": "meshcentral",
          "image": "agent-logo.png"
        }
      }
  },
}
`

I download agent 32/64 bit, but in URL Server string "local":
![изображение](https://user-images.githubusercontent.com/34236345/212067688-f22b81c0-47d1-4349-8e70-77778244d272.png)

**Comment by si458** [comment, marked as answer]
You have lanonly as true and wanonly as true

Lanonly has priority, try setting lanonly to false, then remove agent and reinstall an agent

**Reply by artem33region** [reply]
A lot of thanks!!!
